docs/openstack/create-and-connect-to-the-VM/ssh-to-the-VM.md

```diff
@@ -0,0 +1,320 @@
+# SSH to the VM
+
+**Shell**, or **SSH**, is used to administering and managing Linux workloads.
+Before trying to access instances from the outside world, you need to make sure
+you have followed these steps:
+
+- You followed the instruction in [Create a Key Pair](../access-and-security/create-a-key-pair.md)
+to set up a public ssh key.
+
+- Your public ssh-key has selected (in "Key Pair" tab) while
+[launching the instance](launch-a-VM.md).
+
+- [Assign a Floating IP](assign-a-floating-IP.md) to the instance in order to
+access it from outside world.
+
+- Make sure you have added rules in the
+[Security Groups](../access-and-security/security-groups.md#allowing-ssh) to
+allow **ssh** using Port 22 is opened to the instance.
+
+!!! info "How to update New Security Group(s) on any running VM?"
+    If you want to attach/deattach any new Security Group(s) to/from a running VM
+    after it has launched. First create all new Security Group(s) with all the rules
+    required. Following [this guide](../access-and-security/security-groups.md#update-security-groups-to-a-running-vm),
+    you'll be able to attach any newly created security group(s) with all the
```

Comment:
remove "newly created" it does not matter when they created it they can add any security group.